RecoEgamma/EgammaMCTools/python/pfClusterMatchedToPhotonsSelector_cfi.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+particleFlowClusterECALMatchedToPhotons = cms.EDProducer('PFClusterMatchedToPhotonsSelector',
```

Comment:
This duplicates the config automatically produced by ParameterSetDescription
Please remove this file